pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -40,8 +40,11 @@ let
   pkgSuffix = optionalString (versionOlder version "304") "-pkg0";
   i686bundled = versionAtLeast version "391" && !disable32Bit;
 
-  libPathFor = pkgs: pkgs.lib.makeLibraryPath [ pkgs.libdrm pkgs.xorg.libXext pkgs.xorg.libX11
-    pkgs.xorg.libXv pkgs.xorg.libXrandr pkgs.xorg.libxcb pkgs.zlib pkgs.stdenv.cc.cc pkgs.wayland ];
+  libPathFor = pkgs: pkgs.lib.makeLibraryPath ([
+    pkgs.libdrm pkgs.xorg.libXext pkgs.xorg.libX11
+    pkgs.xorg.libXv pkgs.xorg.libXrandr pkgs.xorg.libxcb pkgs.zlib pkgs.stdenv.cc.cc
+    pkgs.wayland pkgs.mesa
+  ] ++ (if useGLVND then [ pkgs.libglvnd ] else [ pkgs.libGL ]));
```

Comment:
Just to be sure, you are aware that `pkgs.libGL` is just libglvnd stubs on Linux, right?  Doing the dependency directly improves something here, I assume?

Reply:
So libGL is for easier support both linux and darwin? I am confused.

Reply:
@vcunat are you telling me that pkgs.libGL is an alias to libglvnd on Linux or that the libGL source devolves to just being libglvnd itself on Linux or am I not understanding at all?

Reply:
Okay, I read .../mesa/stubs.nix and I get it now.

I see why this is unnecessary and what useGLVND is needed for insider buiilder.sh.

Thanks for the feedback, this had been a small blind spot until now. I'll fix this up and update tomorrow.

Reply:
`libGL` provides building against GL libs, on any platform.  Right now, Linux ones use glvnd and Darwin ones do not.